SwiftMessageBar/SwiftMessageBar.swift

```diff
@@ -19,7 +19,8 @@ public struct MessageBarConfig {
   
   public init(errorColor: UIColor = UIColor.red, successColor: UIColor = .green, infoColor: UIColor = .blue,
               titleColor: UIColor = .white, messageColor: UIColor = .white, statusBarHidden: Bool = false,
-              successIcon: UIImage? = nil, infoIcon: UIImage? = nil, errorIcon: UIImage? = nil) {
+              successIcon: UIImage? = nil, infoIcon: UIImage? = nil, errorIcon: UIImage? = nil,
+              languageDirection: NSLocale.LanguageDirection = .leftToRight) {
```

Comment:
I think using `.unknown` as default language direction makes more sense here, we don't care about it unless it's right to left so the parameter should reflect that